orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -460,13 +460,15 @@ def parse_outputs_for_onnx_export_and_extract_schema(module, inputs, kwargs):
     module.eval()
     output_names = None
     output_dynamic_axes = None
+    deepcopy_flag = True
```

Comment:
Maybe rename to `is_copy` or similar, so it's the meaning of true/false is more clear?